spec/pom.xml

```diff
@@ -34,6 +34,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>se.fortnox.reactivewizard</groupId>
```

Comment:
If included, do we have duplicates of the things it brings with it? mokito etc

Reply:
Yes, but it is managed by the dependency management I think